### PR TITLE
Updating readme

### PR DESCRIPTION
**Pull request by mikedao**
I opened this pull request to update the Readme to reflect the change of purpose of this repository--from a restaurant web application to a multi-tenant micro-finance lending platform.

Other changes:
* Changed format from RDOC format to markdown
* Added information about Turing, the project, and the expectations
* Added the list of collaborators, as well as links to their home github pages.

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+(http://turing.io)
+
+The goal is to take legacy code, namely a website that was previously a site
+for a restaurant, and turn it into a multitenant site.
+
+In particular, we are building Keevahh.
+
+Keevahh
+
+Micro-lending is a powerful tool for social progress. Let’s rework Dinner Dash
```
Comment (EmilyMB) [inline README.md]: Perhaps rephrase? Non-Turing folks won't know what Dinner Dash is. :confused:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+* They can add multiple loans from multiple borrowers to their cart
+* They can then checkout and the funds are allocated to the borrowers
+* The borrowers are notified that funding has come through and their loan
+request page is updated
+* Lending like this is more fun together – it’d be great if lenders could band
+together into lending groups that make loans together on a weekly or monthly basis.
+
+#### On this project:
+
+* Emily Berkeley - https://github.com/EmilyMB
```
Comment (EmilyMB) [inline README.md]: :+1:  Spelled right this time :smile:

File: README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+## The Pivot
+
+This is a project for the Turing School of Software and Design.
+(http://turing.io)
+
+The goal is to take legacy code, namely a website that was previously a site
+for a restaurant, and turn it into a multitenant site.
+
+In particular, we are building Keevahh.
```
Comment (EmilyMB) [inline README.md]: Maybe, "we are building Keevah, a micro-lending platform, mimicking functionality of the Kiva Microfund's site, Kiva.org"?

File: README.md (hunk 1 above)
Comment (mikedao) [inline README.md]: This was copied and pasted from the spec. I kind of explain what it is above.

File: README.md (hunk 3 above)
Comment (mikedao) [inline README.md]: :sparkles:

File: README.md (hunk 1 above)
Comment (EmilyMB) [inline README.md]: Just because it came from Turing doesn't mean it couldn't be improved :wink:

**Pull request merged**